lib/gtfs/helpers.ex

```diff
@@ -1,8 +1,8 @@
-defmodule Helpers do
+defmodule Gtfs.Helpers do
   @doc """
   Apply a function to all of the vaues in a map
 
-  iex> Helpers.map_values(%{a: 1, b: 2}, fn x -> x+1 end)
+  iex> Gtfs.Helpers.map_values(%{a: 1, b: 2}, fn x -> x+1 end)
```

Comment:
No need to change, but you can do `alias Gtfs.Helpers` at the top of `test/gtfs/helpers_test.exs` and then you wouldn't need to update the documentation here. The doctests are generated as tests in the module and inherit any import or aliases in there.